Funcky.Test.Internal/Funcky.Test.Internal.csproj

```
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk; Microsoft.Build.CentralPackageVersions">
+
+    <PropertyGroup>
+        <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
+        <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net4.8</TargetFrameworks>
```

One target framework should be enough, right?
```suggestion
        <TargetFramework>netcoreapp3.1</TargetFramework>
```

---

Needs netstandard2.0 -> Fixed in 63a091c5a8260d8a2caa59c7e6ba7d3a783de0a3